components/brave_wallet/browser/brave_wallet_utils.h

```diff
@@ -11,6 +11,8 @@
 #include <string>
 #include <vector>
 
+#include "base/containers/span.h"
```

Comment:
span seems not referenced in this header

Reply:
Fixed.